test/gtest/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ function(add_gtest TEST_NAME)
   add_dependencies(tests test_${TEST_NAME})
   add_dependencies(check test_${TEST_NAME})
   target_compile_options(test_${TEST_NAME} PRIVATE -Wno-global-constructors -Wno-undef)
-  target_include_directories(test_${TEST_NAME} PRIVATE ../)
+  target_include_directories(test_${TEST_NAME} PRIVATE ../ PRIVATE ${PROJECT_SOURCE_DIR}/driver)
```

Comment:
What code are tests using from the driver

Reply:
#include "activ_driver.hpp" would need this line...